### ISSUE TITLE
How to get instance ids?

### ISSUE THREAD

**Issue by mabushey** [skyscrapers/terraform-neo4j#6]
Hello,

I have your module working great due to your previous help, thank you. :) I would like to be able to get the instance ID's in other terraform code. It seems like I should be able to add a block like this to outputs.tf:
```
output "instance_ids" {
  value = [ "${aws_instance.instance_no_ebs.*.id}" ]
}
```

and then in my code use something like `instance = "${module.neo4j.instance_ids.2}"`
Is this the correct way to do this or is there a better way?

Thanks!

**Comment by MattiasGees** [comment]
Yes that should work.

**Comment by mabushey** [comment]
I get: `Error: output 'instance_ids': unknown resource 'aws_instance.instance_no_ebs' referenced in variable aws_instance.instance_no_ebs.*.id`

**Comment by mabushey** [comment]
from `terraform show`:
```
...
module.neo4j.core.aws_instance.instance_no_ebs.2:                                                                                                                                                      
  id = i-01f04b750b8c64513 
...
```

**Comment by mabushey** [comment]
This works:
outputs.tf:
```
output "core_count" {
  value = "${var.core_count}"
}
```
```
$ terraform console
> module.neo4j.core_count
3
```

**Comment by iuriaranda** [comment]
Hi @mabushey,

First of all I would like to apologise for not coming back to you in time.

I'm not sure if this issue is still relevant for you, but in any case I've added some extra outputs to the module (#7) so there's more information available to the caller (available in release [`0.9.6`](https://github.com/skyscrapers/terraform-neo4j/releases/tag/0.9.6)). You should be able to get the info you need in your Terraform code with:

```tf
${module.neo4j.instance_ids[2]}
```

FYI, the reason you were getting that error is because the resource `aws_instance.instance_no_ebs` is inside the sub-module `core`, so you can't access it directly from the neo4j module. You have to access that info through the sub-module's own outputs 🙂
